astropy/io/ascii/cparser.pyx

```diff
@@ -848,7 +848,8 @@ cdef class FastWriter:
                   fill_values=[],
                   fill_include_names=None,
                   fill_exclude_names=None,
-                  fast_writer=True):
+                  fast_writer=True,
+                  overwrite=False):
```

Comment:
Maybe I'm missing something, but is this necessary?  It looks like `overwrite` is popped off the kwargs early in `ui.write()` and would never make it into `cparser.pyx`.  And `overwrite` is not referenced again anywhere in the fast writer, so that also points to not being necessary.